docs/v0.7/en/reference/sql/create.md

```diff
@@ -106,6 +111,23 @@ CREATE TABLE IF NOT EXISTS temperatures(
 ) engine=mito with(ttl='7d', regions=10, storage="Gcs");
 ```
 
+Create a table whose compaction time window is 1 day:
```

Comment:
Can you explain more about the compaction strategy? For example, the usage scenarios, the advantages, etc.?

Reply:
Added a link to twcs https://cassandra.apache.org/doc/latest/cassandra/managing/operating/compaction/twcs.html